File: op/lab2/python/lab2.py
import math 

X = float(input( "Enter value of the X variable: "))

if X>0 and X<math.pi:
    Y = math.cos(X)
else:
    if X>0:
        Y = -1
    else:
        Y = 1
    
print("Y: ",Y)

File: op/1semester/lab2/c++/lab2.cpp
#include <iostream>
#include <math.h>
using namespace std; 

int main() {
    float X,Y;
    const float PI = 3.14;
    cout << "Enter value of the X variable: ";
    cin >> X;
    if (X<PI && X>0) {
        Y = cos(X);
    } else {
        if (X>=0) {
            Y = -1;
        } else {
            Y = 1;
        }
    }
    cout << "Y: " << Y;
    return 0;
}                                  